libirods_rule_engine_plugin-elasticsearch.cpp

```diff
@@ -124,14 +134,12 @@ namespace
 
 			namespace urls = boost::urls;
 
-			urls::result<urls::url_view> result = urls::parse_uri(host);
-			if (!result) {
+			urls::result<urls::url_view> url_parse_result = urls::parse_uri(host);
+			if (!url_parse_result) {
 				rodsLog(LOG_ERROR, fmt::format("{}: could not parse service URL [{}].", __func__, host).c_str());
 				continue;
 			}
```

Comment:
Following this, check if the port is available. If it isn't, then log an error/warning for the admin.

Reply:
Done.